tests/Silex/Tests/Provider/TwigServiceProviderTest.php

```diff
@@ -116,4 +116,13 @@ public function testFormWithoutCsrf()
 
         $this->assertInstanceOf('Twig_Environment', $app['twig']);
     }
+
+    public function testServices()
+    {
+        $app = new Application();
+        $app->register(new TwigServiceProvider());
+        foreach ($app->keys() as $id) {
+            $app[$id];
```

Comment:
This test does not test anything useful as services are instantiated only on-demand.